steward/src/cork.rs

```diff
@@ -131,6 +131,13 @@ fn get_encoded_call(request: SubmitRequest) -> Result<Vec<u8>, Error> {
 
             cellar_v2::get_encoded_call(call.function.unwrap(), request.cellar_id)
         }
+        CellarV22(call) => {
```

Comment:
Is there any kind of separator we can use here? It makes theoretical semver clashes very possible.